scripts/lib/wic/canned-wks/raspberrypi.wks

```diff
@@ -0,0 +1,8 @@
+# short-description: Create Raspberry Pi SD card image
+# long-description: Creates a partitioned SD card image for use with
+# Raspberry Pi. Boot files are located in the first vfat partition.
+
+part /boot --source bootimg-partition --ondisk mmcblk0 --fstype=vfat --label boot --active --align 4096 --size 20
+part / --source rootfs --ondisk mmcblk0 --fstype=ext4 --label platform1 --align 4096 --uuid b488d900-9cc0-11e9-8833-cf4cb425a04d
+part / --source rootfs --ondisk mmcblk0 --fstype=ext4 --label platform2 --align 4096 --uuid a2d8ad56-9c9e-11e9-afd8-07ffb5f9d79e
+part /media --ondisk mmcblk0 --size 1024 --fstype=ext4 --label media --align 4096 --uuid 79664834-9c9e-11e9-9603-a744b9801557
```

Comment:
If i separate this in two commits, this will be the only line added to the next commit. I think it makes sense to have it the /media partition included in this commit as well.

Reply:
@sashko